WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -292,6 +294,11 @@ private void handleOpenPageIntent(Intent intent) {
                     break;
                 case ARG_ME:
                     mBottomNav.setCurrentPosition(PAGE_ME);
+                    if (intent.getBooleanExtra(WPMainActivity.ARG_SHOW_ZENDESK_HINT_SNACKBAR, false)) {
+                        Snackbar.make(findViewById(R.id.coordinator),
+                                R.string.support_push_notification_message_hint_snackbar, Snackbar.LENGTH_LONG).show();
```

Comment:
Can I suggest to use a timeout of `2 x  Snackbar.LENGTH_LONG` ?
Since there is no way to bring it back when it's gone, and the users can still don't know where to go to open the tickets. (feel free to ignore this if the current timeout on real usage is long enough).

Reply:
addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/8043/commits/998274e91c2ebddcf6be460ad0d23f4c72a99c10